xzless: add examples (#10179)
tldr-pages/tldr: pages/common/xzless.md

> See also: `less`.
> More information: <https://manned.org/xzless>.

- View a compressed file:

> `xzless {{path/to/archive.xz}}`

Reviewer: `xzless` also supports `lzma` compressed archives.
```suggestion
`xzless {{path/to/archive}}`
```

> - View a compressed file [r]ecursively page by page:
> 
> `xzless -r {{path/to/archive.xz}}`
> 

Reviewer: From `man less`:
> -r or --raw-control-chars
> Causes  "raw"  control  characters to be displayed.  The default is to display control characters using the caret notation; for example, a control-A (octal 001) is displayed as "^A" (with some exceptions as described under the -U option).
> Warning: when the -r option is  used, less cannot keep track of the actual appearance of the screen (since this depends on how the screen responds to each type of control character). Thus, various display problems may result, such as long lines being split in the wrong place.
> USE OF THE -r OPTION IS NOT RECOMMENDED.
```suggestion
```

> - View a compressed file and display the line [n]umber:

Reviewer: ```suggestion
- View a compressed file and display line numbers:
```

> `xzless -N {{path/to/archive.xz}}`

Reviewer: https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax
```suggestion
`xzless --LINE-NUMBERS {{path/to/archive}}`
```

> - View a compressed file and [f]ollow it:

Reviewer: From `man less`:
> -F or --quit-if-one-screen
> Causes less to automatically exit if the entire file can be displayed on the first screen.

```suggestion
- View a compressed file and quit if the entire file can be displayed on the first screen:
```

> `xzless -F {{path/to/archive.xz}}`

Reviewer: ```suggestion
`xzless --quit-if-one-screen {{path/to/archive}}`
```

> `xzless -F {{path/to/archive.xz}}`

Reviewer: ```suggestion
`xzless --quit-if-one-screen {{path/to/archive}}`
```
